google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -84,7 +84,50 @@ public boolean isSupportAlreadyAdded(@NotNull Module module,
         || !facetsProvider.getFacetsByType(module, AppEngineStandardFacet.ID).isEmpty();
   }
 
-  static class AppEngineFlexibleSupportConfigurable extends FrameworkSupportInModuleConfigurable {
+  /** Initializes the Flexible facet by settings the default paths for app.yaml and Dockerfile
+   * and generating the necessary run configurations.
+   */
+  public void setupFacet(@NotNull AppEngineFlexibleFacet facet,
```

Comment:
I just factored this into a new method. Code should be unchanged.

What do you think of turning this into a static method? So we don't have to instantiate AppEngineFlexibleSupportProvider if we want to call this from the outside.